docs/02-membership.md

```diff
@@ -0,0 +1,350 @@
+**Core Contributor** - "The Ethereum Protocol needs careful and long-term thinking. Despite core-devs are very value driven, their short lifetime (compared to the pretended Ethereum lifetime) introduces some economic opportunity costs which can put Ethereum development at risk. Funding is just a problem that gets in the way between the protocol development and core-devs. I wish PG can fix this problem!" <br><br>
+**Core Contributor** - "The Guild is a great way to assure the high quality of contributions while diversifying the set of contributors, leading to solid and always-improving protocols for the whole ecosystem." <br><br>
+**Core Contributor** - "The Guild will help retain existing and attract new talented contributors to the Ethereum revolution." <br><br>
+**Core Contributor** - "The Protocol Guild plays a vital role in expanding the Ethereum ecosystem by attracting skilled individuals to the blockchain space. By rewarding highly engaged participants, it provides an additional incentive to drive continuous improvement. Furthermore, the Guild fosters collaboration among key stakeholders within Ethereum, facilitating crucial discussions on pressing topics. This collective effort ultimately unites contributors under a shared objective: to enhance the decentralized world for the better." <br><br>
+**Core Contributor** - "The whole is greater than the sum of the parts.  But without the parts there is no whole. This project addresses the parts without diminishing the whole." <br><br>
+**Core Contributor** - "This type of a public funding should be main source of funds for such a projects like Ethereum blockchain research and development." <br><br>
+**Core Contributor** - "With its rewards beyond casual wages or project grants, Protocol Guild inspires members to apply their efforts to the Ethereum development in unrehearsed ways and directions where forces of EF and big companies are missed. It preserves the strong community engagement over major pathways, which safeguards Ethereum's decentralization." <br><br>
+**danceratopz (Testing (EF))** - "I believe the Protocol Guild will play an important role in helping ensure a sustainable future for Ethereum as a foundational public good. Its infrastructure enables Ethereum projects to channel funds into the protocol's continuous enhancement, thereby incentivizing and helping retain the dedicated individuals who work on its development. This system of support allows Ethereum's growth without reliance on a single or a collective group of entities. It even paves the way for core developers to operate independently of any organization, sustained by Guild-facilitated funding." <br><br>
+**Danny Ryan (Consensus R&D (EF))** - "Public goods are hard." <br><br>
```

Comment:
I'd move the anon comments below all the non-anon ones, even perhaps in their own section. Otherwise, it feels like these are hidden.